content/videos/challenges/113-4d-hypercube-aka-tesseract/index.json

```diff
@@ -0,0 +1,101 @@
+{
+    "title": "4D Hypercube (aka 'Tesseract')",
+    "description": "In this coding challenge, I visualize a 4D Hypercube (aka \u201cTesseract\u201d) in Processing (Java).",
+    "videoNumber": "113",
+    "videoId": "XE3YDVdQSPo",
+    "date": "2018-08-22",
+    "languages": ["Processing"],
+    "topics": [
+        "3d rendering",
+        "orthographic projection",
+        "rotation matrix",
+        "4d hypercube",
+        "tesseract"
+    ],
+    "canContribute": true,
+    "relatedChallenges": [],
```

Comment:
@kfahn22 we should add the 2D projection challenge here! (See #262). We could also consider doing a pass over all videos once all the challenges are ported.